markdowns/20_REST_API.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: tutorial
+colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/20_REST_API.ipynb
+toc: True
+title: "Haystack with REST API"
+last_updated: 2022-11-23
+level: "advanced"
+weight: 115
+description: Create a production-ready pipeline and interact with REST API.
+category: "QA"
+aliases: ['/tutorials/rest-api']
+---
+    
+
+# Tutorial: Using Haystack with REST API
+
+- **Level**: Advanced
+- **Time to complete**: 30 minutes
+- **Prerequisites**: Basic understanding of Docker, basic knowledge of Haystack pipelines 
+- **Nodes Used**: `ElasticsearchDocumentStore`, `EmbeddingRetriever`
+- **Goal**: Learn how you can interact with Haystack through REST API.
+
+## Overview
+
+Haystack enables you to apply the latest NLP technology to your own data and create production-ready applications. Building an end-to-end NLP application requires the combination of multiple concepts. Here are those consepts:
+* **DocumentStore** stores the data. You will use Elasticsearch for this tutorial.
+* **Haystack** pipelines convert documents, index them to DocumentStore, and run NLP tasks such as question answering and document search.
```

Comment:
convert files in to **Documents** or Haystack **Documents** --> just because this might be a good opportunity for an aha moment that the learner gets about what their files are processed into